packages/elements/src/clock/__test__/clock.interactive.test.js

```diff
@@ -39,75 +47,79 @@ describe('clock/Interactive', () => {
       elem.dispatchEvent(keyboardEvent('keyup', keyOption));
     };
 
+    const updateBtnSelector = () => {
+      incrementBtnInHours = hoursSegment.querySelector('[part=increment-button]');
+      decrementBtnInHours = hoursSegment.querySelector('[part=decrement-button]');
+      incrementBtnInMinutes = minutesSegment.querySelector('[part=increment-button]');
+      decrementBtnInMinutes = minutesSegment.querySelector('[part=decrement-button]');
+    }
+
     beforeEach(async () => {
       el = await fixture('<ef-clock interactive></ef-clock>');
       hoursSegment = el.shadowRoot.querySelector('[part~=hours]');
       minutesSegment = el.shadowRoot.querySelector('[part~=minutes]');
+      updateBtnSelector();
     });
 
     it('Increases hour when increase button is clicked', async () => {
-      const incrementBtn = hoursSegment.querySelector('[part=increment-button]');
-
       await onTapstart(hoursSegment, el);
-      await onTapstart(incrementBtn, el);
+      updateBtnSelector();
```

Comment:
Why do you need to call this function again when you already called it in `forEach` ?